starter-code/functions-and-arrays.js

```diff
@@ -43,7 +62,11 @@ console.log(average);
 
 // Array of Strings
 function averageWordLength (array) {
```

Comment:
This is the fastest solution. Another would be to create an array of word lengths and then apply `averageNumbers`on it.